customers/services/profile.py

```diff
@@ -131,7 +144,20 @@ def parse_user_edge(self, gql_edge: Dict[str, dict]) -> HelsinkiProfileUser:
     def parse_user(self, profile: Dict[str, dict]) -> HelsinkiProfileUser:
         user_id = from_global_id(profile.pop("id"))
         email = (profile.pop("primary_email") or {}).get("email")
-        return HelsinkiProfileUser(UUID(user_id), email=email, **profile)
+
+        primary_address = profile.pop("primary_address")
+        address = (primary_address or {}).get("address")
+        postal_code = (primary_address or {}).get("postal_code")
+        city = (primary_address or {}).get("city")
```

Comment:
Would it be a bit simpler to do
```suggestion
        primary_address = profile.pop("primary_address") or {}
        address = primary_address.get("address", "")
        postal_code = primary_address.get("postal_code", "")
        city = primary_address.get("city", "")
```

Reply:
`profile.pop("primary_address")` without default value throws error. cleaned this thing up a bit.